css/components/modal.css

```diff
@@ -0,0 +1,38 @@
+.modal {
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 25;
```

Comment:
Not a fan of arbitrary numbers for z-index. Analyze your layout well and assign the accurate value.